src/java/com/wolfssl/provider/jsse/WolfSSLEngineHelper.java

```diff
@@ -1121,19 +1121,43 @@ private void setLocalSigAlgorithms() {
             /* Get restricted signature algorithms for ClientHello if set by
              * user in "wolfjsse.enabledSigAlgorithms" Security property */
             String sigAlgos = WolfSSLUtil.getSignatureAlgorithms();
-
-            if (sigAlgos != null) {
-                ret = this.ssl.setSignatureAlgorithms(sigAlgos);
-                if (ret != WolfSSL.SSL_SUCCESS &&
-                    ret != WolfSSL.NOT_COMPILED_IN) {
-                    WolfSSLDebug.log(getClass(), WolfSSLDebug.INFO,
-                        () -> "error restricting signature algorithms based " +
-                        "on wolfjsse.enabledSignatureAlgorithms property");
-                } else if (ret == WolfSSL.SSL_SUCCESS) {
-                    WolfSSLDebug.log(getClass(), WolfSSLDebug.INFO,
-                        () -> "restricted signature algorithms based on " +
-                        "wolfjsse.enabledSignatureAlgorithms property");
-                }
+            String sigSchemes =
+                   WolfSSLUtil.getSignatureSchemes(this.clientMode);
+            String cleanSigSchemes =
+                   WolfSSLUtil.formatSigSchemes(sigAlgos, sigSchemes);
+            
+            if (cleanSigSchemes != null)
```

Comment:
Could you add braces around this `if`, for consistency, ie:

```
if (cleanSigSchemes != null) {
    ret = this.ssl.setSignatureAlgorithms(cleanSigSchemes);
}

if (ret != WolfSSL.SSL_SUCCESS && ...
```

Also, if `cleanSigSchemes == null`, is the default `ret` value handled ok in the following if/else?